archinstall/lib/installer.py

```diff
@@ -734,61 +734,72 @@ def _add_systemd_bootloader(self, root_partition: disk.PartitionModification):
 		entries_dir = loader_dir / 'entries'
 		entries_dir.mkdir(parents=True, exist_ok=True)
 
+		comments = (
+			'# Created by: archinstall\n',
+			f'# Created on: {self.init_time}\n'
+		)
+
+		microcode = []
+
+		if not SysInfo.is_vm():
+			vendor = SysInfo.cpu_vendor()
+			if vendor == "AuthenticAMD":
+				microcode.append('initrd  /amd-ucode.img\n')
+			elif vendor == "GenuineIntel":
+				microcode.append('initrd  /intel-ucode.img\n')
+			else:
+				debug(
+					f"Unknown CPU vendor '{vendor}' detected.",
+					"Archinstall won't add any ucode to systemd-boot config.",
+				)
+
+		# blkid doesn't trigger on loopback devices really well,
+		# so we'll use the old manual method until we get that sorted out.
+
+		options_entry = f'rw rootfstype={root_partition.safe_fs_type.fs_type_mount} {" ".join(self._kernel_params)}'
+
+		for sub_vol in root_partition.btrfs_subvols:
+			if sub_vol.is_root():
+				options_entry = f"rootflags=subvol={sub_vol.name} " + options_entry
```

Comment:
Very minor question, but would it make sense to add a `break` here?

Reply:
I think so.